src/main/java/mods/eln/sixnode/TreeResinCollector/TreeResinCollectorDescriptor.java

```diff
@@ -52,7 +51,7 @@ void draw(float factor) {
 
     @Override
     public boolean use2DIcon() {
```

Comment:
I can remove it already for GenericItemBlockUsingDamageDescriptor subclasses.

Some items (GenericItemUsingDamageDescriptor subclasses) do still return false in order to do some custom rendering. Have to address them later.